homeassistant/components/decora_wifi/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "step": {
+            "user": {
+                "title": "myLeviton Login",
+                "description": "Enter your myLeviton login information.",
+                "data": {
+                    "username": "[%key:common::config_flow::data::username%]",
+                    "password": "[%key:common::config_flow::data::password%]"
+                }
+            },
+            "reauth": {
+                "title": "myLeviton Login",
```

Comment:
you can use keys for this as well referring to the user title, this to make sure there is less to translate.